tests/EdFi.Admin.DataAccess.IntegrationTests/Contexts/PostgreSQLUserContextFactoryTests.cs

```diff
@@ -5,43 +5,271 @@
 using System.Linq;
-using EdFi.Admin.DataAccess.DbConfigurations;
+using System.Transactions;
+using EdFi.TestFixture;
+using Microsoft.EntityFrameworkCore;
 
 namespace EdFi.Ods.Admin.DataAccess.IntegrationTests.Contexts
 {
+
     [TestFixture]
     public class PostgreSQLUserContextFactoryTests
     {
-        [Test, Explicit]
-        public void Given_configured_for_Postgres_then_create_PostgresUsersContext_make_a_change_and_save_it()
+        private PostgresUsersContext _context;
+        private TransactionScope _transaction;
+
+        [SetUp]
+        public void Setup()
+        {
+            AppContext.SetSwitch("Npgsql.EnableLegacyTimestampBehavior", true);
+
+            var builder = new ConfigurationBuilder()
+               .AddJsonFile($"appSettings.json", true, true)
+               .AddJsonFile($"appSettings.development.json", true, true);
+
+            var config = builder.Build();
+            var connectionString = config.GetConnectionString("PostgreSQL");
+
+            var optionsBuilder = new DbContextOptionsBuilder();
+            optionsBuilder.UseNpgsql(connectionString);
+            _context = new PostgresUsersContext(optionsBuilder.Options);
+
+            _transaction = new TransactionScope();
+        }
+
+        [TearDown]
+        public void Teardown()
         {
-            var connectionStringsProvider = A.Fake<IAdminDatabaseConnectionStringProvider>();
-            A.CallTo(() => connectionStringsProvider.GetConnectionString()).Returns("Host=localhost; Port=5432; Username=postgres; Database=EdFi_Admin_Test; Application Name=EdFi.Ods.WebApi;");
+            _transaction.Dispose();
+        }
 
-            DbConfiguration.SetConfiguration(new DatabaseEngineDbConfiguration(DatabaseEngine.Postgres));
-            var context = new UsersContextFactory(connectionStringsProvider, DatabaseEngine.Postgres)
-                .CreateContext();
+        [Test]
+        public void Given_configured_for_Postgres_then_create_PostgresUsersContext_make_a_change_and_save_it()
+        {
 
             var testVendor = new Vendor()
             {
                 VendorName = "Test"
             };
 
-            int originalCount = context.Vendors.Count();
-            context.Vendors.Add(testVendor);
-            context.SaveChanges();
-            context.Vendors.Count().ShouldBe(originalCount + 1);
-            context.Vendors.Remove(testVendor);
-            context.SaveChanges();
-            context.Vendors.Count().ShouldBe(originalCount);
+            int originalCount = _context.Vendors.Count();
+            _context.Vendors.Add(testVendor);
+            _context.SaveChanges();
+            _context.Vendors.Count().ShouldBe(originalCount + 1);
+            _context.Vendors.Remove(testVendor);
+            _context.SaveChanges();
+            _context.Vendors.Count().ShouldBe(originalCount);
+        }
+
+        [TestFixture]
+        public class When_creating_a_user : PostgreSQLUserContextFactoryTests
```

Comment:
I added some more tests to make sure  that we're doing the `OnModel` setup correctly in the PostgreSQL context.